README.md

```diff
@@ -140,7 +140,7 @@ If your URL needs a more elaborate request, it's better if you perform it separa
 
 ## Contributors ✨
 
-Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
+Thanks go to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
```

Comment:
Correct, but I guess it's All Contributors' fault ahahah
https://github.com/all-contributors/all-contributors#contributors-